put includes out of extern "C"

fix the error: template with C linkage when compiling clara/region

--- include/libregionMap.h.orig	2011-07-28 08:40:34.000000000 +0200
+++ include/libregionMap.h	2013-09-11 12:32:04.948027848 +0200
@@ -1,16 +1,16 @@
 #ifndef _LIB_REGION_MAP_H
 #define _LIB_REGION_MAP_H
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-  
 /* Includes from other libraries  */
 #include "t3d/t3d.h"
 #include "libimages3d.h"
 #include "libdtm.h"
 #include "libdtm_gdhe.h"
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
 /* Structure propres à la lib */
 
 #define RMAP_MAX_LINES		(1000)
